spec.bs

```diff
@@ -319,30 +387,72 @@ Note: A later algorithm [=asserts=] that, for any [=contribution cache entry=]
     before the [=process contributions for a batching scope=] steps are
     performed given the entry's [=contribution cache entry/batching scope=].
 
-Overriding `contributeToHistogramOnEvent()` processing {#contributetohistogramonevent}
---------------------------------------------------------------------------------------
+Overriding `contributeToHistogramOnEvent()` processing {#overriding-contributetohistogramonevent}
+-------------------------------------------------------------------------------------------------
 
 Each API may also set the [=PrivateAggregation/should perform default
 contributeToHistogramOnEvent() processing=] algorithm for the
 {{PrivateAggregation}} object it exposes. This hook enables the embedding API to
 override processing for any {{PrivateAggregation/
 contributeToHistogramOnEvent()}} call it wants to.
 
-This algorithm should return true to indicate that the regular processing
+This hook algorithm should return true to indicate that the regular processing
 defined in this spec should occur (for that call). It should return an
 [=exception=] to indicate an error has occurred in the embedding API's
 processing. Alternatively, it should return false to indicate that the embedding
 API will process the call, but no [=exception=] should be [=exception/thrown=].
 
-If the embedding API overrides processing for a call (i.e. the algorithm does
-not return true), it should  accept any `event` that is a [=string/
-concatenation=] of « "<code>reserved.</code>", |errorEvent| » where |errorEvent|
-is an [=internal error event=] and should accept any `contribution` with a
-[=converted to a JavaScript value|JavaScript value=] that is [=converted to an
-IDL value|convertible=] to the IDL type {{PAHistogramContribution}}. That is, it
-should not return an [=exception=] in those cases. However, the embedding API
-may accept additional `event`s or `contribution`s that would not be accepted by
-the default processing defined in this spec.
+<table class="complex data longlastcol">
```

Comment:
The table's not strictly necessary, but I thought it's kind of nice for clarity.